tests/plugins/wysiwygarea/manual/detached.html

```diff
@@ -0,0 +1,48 @@
+<button id="toggleButton1">Toggle</button>
+<button id="setButton">Set data</button>
+
+<div>
+	<div id="editor">
+		<p>Lorem ipsum dolor sit amet.</p>
+	</div>
+</div>
+
+
+<div id="fakedId"></div>
+<div id="fakedId1"></div>
+
+
+<script>
+	var editorElement = document.getElementById("editor");
+
+	var textArea = document.createElement("textarea");
+
+	editorElement.appendChild(textArea);
```

Comment:
Is it the same as in https://github.com/ckeditor/ckeditor4/pull/4463#discussion_r551381519?

Reply:
It's the same case. [https://github.com/ckeditor/ckeditor4/pull/4463#discussion_r551806398](https://github.com/ckeditor/ckeditor4/pull/4463#discussion_r551806398) :smile: